src/ExpressionParser.php

```diff
@@ -689,20 +689,24 @@ public function parseFilterExpressionRaw($node)
     /**
      * Parses arguments.
      *
-     * @param bool $namedArguments Whether to allow named arguments or not
-     * @param bool $definition     Whether we are parsing arguments for a function (or macro) definition
-     *
      * @return Node
      *
      * @throws SyntaxError
      */
-    public function parseArguments($namedArguments = false, $definition = false)
+    public function parseArguments()
     {
+        $namedArguments = true;
```

Comment:
this changes the default value, which could be unexpected if some places where reusing this without expecting named arguments.

Reply:
Fixed